packages/itwinui-react/src/styles.js/vite.config.mjs

```diff
@@ -86,6 +87,21 @@ function postcssAddIuiVersion() {
   );
 }
 
+// https://github.com/vitejs/vite/pull/16051#issuecomment-2076616816
+function cssModuleSideEffectful() {
+  return {
+    name: 'css-module-side-effectful',
+    enforce: 'post',
+    transform(_, id) {
+      if (id.endsWith('.module.css')) {
+        return {
+          moduleSideEffects: 'no-treeshake',
+        };
+      }
+    },
+  };
+}
```

Comment:
Nice find. My only suggestion would be to inline this plugin in the config above, so that we get better types.